japanese_address_fields.html

```diff
@@ -25,13 +23,18 @@
     <input id="id_address2" class="" type="text" name="address2">
   </div>
 {% else %}
-  <div class="input-select ak-input-type-user input-field-group ak-err-below">
+  <div class="input-text ak-input-type-user input-field-group ak-err-below">
     <label for="id_city">{% filter ak_text:"field_city" %}City{% endfilter %}</label>
-    {% include "./japanese_city_select.html" %}
+    <input id="id_city" class="" type="text" name="city">
   </div>
   <div class="input-text ak-input-type-user input-field-group ak-err-below">
-    <label for="id_address1">{% filter ak_text:"field_billing_address" %}Billing
+    <label for="id_address1">{% filter ak_text:"field_street_address" %}Street
       address{% endfilter %}</label>
     <input id="id_address1" class="" type="text" name="address1">
   </div>
+  <div class="input-text ak-input-type-user input-field-group ak-err-below">
+    <label for="id_address2">{% filter ak_text:"field_billing_address" %}Billing
```

Comment:
i think maybe this is probably more appropriate to be "field_building_name" or "field_street_address_2" or even "field_billing_address_2" while the previous field might be "field_billing_address_1"? But i suppose this just has to match the ak translation id. So perhaps this is something Rachel would be able to confirm.

Reply:
I think that makes sense - I am just hesitant to change up the names for this since I think translations have already been added based on these `ak_text` values. I might update this in a follow up PR depending on what Rachel says in the morning, if that's ok?

Reply:
thanks for the flag here! I'm planning to collate/review all the translation strings for our donate pages soonish, so we can wait until then to standardise the labels I think